src/components/ArticleCard.jsx

```diff
@@ -51,27 +59,10 @@ export default function ArticleCard() {
             {article.comment_count} Comments
           </Link>
         </p>
-        <div className="votes">
-          <p
-            onClick={() => {
-              setVotes((currVotes) => {
-                return currVotes + 1;
-              });
-            }}
-          >
-            👍🏼
-          </p>{" "}
-          {article.votes}
-          <p
-            onClick={() => {
-              setVotes((currVotes) => {
-                return currVotes + 1;
-              });
-            }}
-          >
-            👎🏼
-          </p>
-        </div>
+      </div>
+      <div className="votes">
+        <p onClick={() => handleClick("1")}>👍🏼</p> {votes}
```

Comment:
Could we use a more semantic html tag than a p tag here for a clickable ui element?